app/models/like.rb

```diff
@@ -1,5 +1,12 @@
 class Like < ApplicationRecord
   # Associations
-  belongs_to :author, class_name: 'User'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
   belongs_to :post
+
+  # Methods
+  after_create :update_post_likes_counter
+
+  def update_post_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
- [ ] Just like with the `Post` model, you should rather use after_save and make the `update_post_likes_counter` a private method.